sources/packages/web/src/views/student/NoticeOfAssessment.vue

```diff
@@ -1,40 +1,59 @@
 <template>
-  <Card class="p-m-4">
-    <template #content>
-      <formio formName="noticeofassessment" :data="initialData"></formio>
-    </template>
-  </Card>
-  <v-btn
-    color="primary"
-    class="p-button-raised ml-2 float-right"
-    @click="confirmAssessment()"
-  >
-    <v-icon size="25">mdi-text-box-plus</v-icon>
-    Confirmation of Assessment
-  </v-btn>
+  <v-container>
+    <div class="mb-4">
+      <header-navigator
+        title="Application details"
+        subTitle="Notice of Assessment"
+        :routeLocation="{
+          name: StudentRoutesConst.STUDENT_APPLICATION_DETAILS,
+          params: {
+            id: applicationId,
+          },
+        }"
+      />
+    </div>
+    <full-page-container>
```

Comment:
The full-page-container itself has a v-container inside. So I would recommend end the v-container starting at line 2 at line 15.  Otherwise fullpage container could be more shrinked.

Reply:
Adjusted!